crates/forge/tests/it/config.rs

```diff
@@ -209,47 +205,11 @@ pub async fn runner_with_config_and_zk(mut config: Config) -> MultiContractRunne
     let output = COMPILED.clone();
     let zk_output = COMPILED_ZK.clone();
 
+    // let project = &*PROJECT;
+    let mut paths = ProjectPathsConfig::builder().root(TESTDATA).sources(TESTDATA).build().unwrap();
+    paths.zksync_artifacts = format!("{TESTDATA}/zkout").into();
```

Comment:
```suggestion
    let paths = ProjectPathsConfig::builder().root(TESTDATA).sources(TESTDATA).artifacts(format!("{TESTDATA}/zkout")).build().unwrap();
```

Reply:
I think we will actually need `zksync_artifacts` and not `artifacts` to be set to that path and I didn't add the builder method for it. Wanted to wait until `4.0` to see how we end up handling multiple compilers with the compiler abstraction.